required/latex-lab/latex-lab-sec.dtx

```diff
@@ -437,45 +437,48 @@
 %    \end{macrocode}
 % This redefines \cs{part} in article class.
 %    \begin{macrocode}
-     \RenewDocumentCommand\part{ s O{#3} m }
+     \@ifundefined{part}{}
       {
-        \if@noskipsec \leavevmode \fi
-        \par
-        \addvspace{4ex}%
-        \@afterindentfalse
+        \RenewDocumentCommand\part{ s O{#3} m }
+         {
+           \if@noskipsec \leavevmode \fi
+           \par
+           \addvspace{4ex}%
+           \@afterindentfalse
 %    \end{macrocode}
 % This are the tagging commands needed at the begin. They open a Part structure
 % and the structure for the title of the heading.
 %    \begin{macrocode}
         % tagging start commands
-        \tag_tool:n {sec-start-part=#2}
+          \tag_tool:n {sec-start-part=#2}
         % end tagging start commands
 %    \end{macrocode}
 % This adds a  manual target if the part is unnumbered or starred.
 % It replaces the hyperref patches.
 %    \begin{macrocode}
-        \bool_lazy_any:nT  
-          {
-            { #1 } 
-            { 
-              \int_compare_p:nNn {\c@secnumdepth}<{-1} 
-            } 
-          }  
-          {
-            \MakeLinkTarget[part]{}
-          }
+           \bool_lazy_any:nT  
+            {
+              { #1 } 
+              { 
```

Comment:
the file seems to have tabs not spaces here and elsewhere, maybe better to change that (I can do this easily in emacs)

Reply:
spaces and tabs are the bane of my life ;-). I deactivated the "space cleaning" facility of my editor as people complained that changes created too large diffs ... but perhaps I should simply reactivate it. (I have no idea where the tabs are from as I never use tab.)